packages/vm/src/vm.ts

```diff
@@ -31,7 +31,7 @@ import type { StateManager } from '@ethereumjs/statemanager'
  *
  * This class is an AsyncEventEmitter, please consult the README to learn how to use it.
  */
-export class VM extends AsyncEventEmitter<VMEvents> {
+export class VM<EVMType extends EVMInterface = EVM, EEIType extends EEIInterface = EEI> {
```

Comment:
So I have now changed `extends EVMInterface = EVMInterface, EEIType extends EEIInterface = EEIInterface` to now use `EVM` and `EEI`. However now I get errors down below on types (see comment) and also tests now throw:

```
tests/api/buildBlock.spec.ts:307:22 - error TS2345: Argument of type 'VM<EVMInterface, EEIInterface>' is not assignable to parameter of type 'VM<EVM, EEI>'.

307     await setBalance(vm, address)
```

I have no idea what causes this..?